brainrender_napari/widgets/atlas_manager_view.py

```diff
@@ -47,9 +58,15 @@ def __init__(self, parent: QWidget = None):
         self.setSelectionMode(QTableView.SelectionMode.SingleSelection)
 
         self.doubleClicked.connect(self._on_row_double_clicked)
-        self.hideColumn(
-            self.model().column_headers.index("Raw name")
-        )  # hide raw name
+        self.hidden_columns = ["Raw name"]  # hide raw name
+        for col in self.hidden_columns:
+            self.hideColumn(self.table.column_headers.index(col))
+
+    def _apply_filters(self, query: str):
+        """Filters the table view based on the query and
+        proxy's internal state."""
+        self.proxy.setFilterFixedString(query)
+        return
```

Comment:
This function needs to be called from `AtlasManagerFilter` so should not start with `_` - `_` denotes private functionality by convention. I would remove this function entirely, and just call `atlas_manager_view.proxy.setFilterFixedString` directly in `AtlasManagerFilter`?

This would remove an unnecessary layer of abstraction.